README.md

```diff
@@ -1 +1,10 @@
-# virtual-schema-installer
+# Virtual Schema Installer
+
```

Comment:
maybe we can add a small description of the project, what-it-does/what-is-it-for

Reply:
I've created a ticket for it: https://github.com/exasol/virtual-schema-installer/issues/5